bigquery/deploy/README.md

```diff
@@ -56,16 +56,28 @@
     ```
 
 * Run indexer on GKE
+  * If you did the "Run Elasticsearch on GKE" step a while ago, you can run
+    these commands to see if `gcloud` and `kubectl` are configured correctly.
+    ```
+    gcloud config get-value project
```

Comment:
I don't know if this section is really necessary, but if we're going to have it we probably can just say "Refresh your default project and credentials if needed: <commands>". In the time it would have taken to check the value of your default project and credentials you could have already set them

Reply:
Done